docs/source/howto/lcf.rst

```diff
@@ -0,0 +1,467 @@
++ SWIR 2             + Q25, Q50, Q75                                  +
++--------------------+------------------------------------------------+
++ Tass. Cap Green.   + Q90, STD                                       +
++--------------------+------------------------------------------------+
++ NDVI               + Q90, STD                                       +
++--------------------+------------------------------------------------+
+
+The selection of these features was shown to result in accurate land cover fraction maps in previous studies (e.g., `Schug et al. (2020) <https://doi.org/10.1016/j.rse.2020.111810>`_). It is, however, possible that mapping land cover fractions in world regions with different phenological regimes or land cover characteristics might require different feature combinations.
+
+We use 
+
+.. code-block:: bash
+
+	force-higher-level parameterfiles/20_lcf_stm_reflectance.prm
+	force-higher-level parameterfiles/20_lcf_stm_vegetation.prm
+
+to compute spectral-temporal metrics of reflectance and vegetation components.
+
+You can access the parameter files `here <../_static/parameter-files/tutorials/lcf/20_lcf_stm_reflectance.prm>`_ and `here <../_static/parameter-files/tutorials/lcf/20_lcf_stm_vegetation.prm>`_ or use the ones provided in the data repository.
+
+.. figure:: img/tutorial-lcf-stm.jpg
+   :height: 330
+
+   *First, second, and third quartile of reflectance of all blue (A) and near infrared (B) observations. 90th Quantile and standard deviation (STD) of Tasseled Cap Grenness (TCG) of all observations (C). All stretches contain 96% of the values (2% - 98%).* |copy| *Franz Schug*
+
+Tip
+
+Please refer to the Spectral Temporal Metrics `tutorial <https://force-eo.readthedocs.io/en/latest/howto/stm.html>`_ for further information about generating spectral-temporal metrics using the Time Series Analysis (TSA) submodule of the FORCE Higher Level Processing system (HLPS). Here, the concept of spectral-temporal metrics is explained and illustrated.
+
+
+Sampling
+-----------------------------------
+
+Land cover fraction mapping with synthetically mixed training data requires spectral information from pure land cover surfaces to begin with, i.e., spectral reference data that can eventually be used to train a model that understands inter- and intra-class spectral variability. 
+
+Here, we are using reference information directly from the imagery (as opposed to, e.g., data from external spectral libraries or ground sampling). We identified 388 reference surfaces, i.e., pixels of 10x10 m that cover only a single land cover type, for five classes: Built-up surfaces (182 reference points), woody vegetation (70), non-woody vegetation (98), bare soil (15), and water (23).
+
+The number of reference points per class varies based on spectral intra-class variability (which is, e.g., higher for non-woody vegetation than for vegetation) and surface availability (e.g., few available reference points for bare soil).
```

Comment:
higher for non-woody vegetation than for vegetation.

is the 2x vegetation really what you mean?